lib/Conversion/TritonGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -1230,17 +1357,19 @@ struct FpToFpOpConversion
         // F8 -> F16
         {{F8E4M3B15TyID, F16TyID}, Fp8E4M3B15_to_Fp16},
         {{F8E4M3FNTyID, F16TyID}, Fp8E4M3B15x4_to_Fp16},
```

Comment:
I think this line should be:
{{F8E4M3FNTyID, F16TyID}, Fp8E4M3_to_Fp16},

This is to keep the NV e4m3 type.